appveyor.yml

```diff
@@ -45,26 +45,26 @@ cache:
 install:
   - if %TARGET:*-msvc=msvc%==msvc set BUILD_ENV=msvc
   - if %TARGET:*-gnu=gnu%==gnu set BUILD_ENV=gnu
-  - ps: 'if ($env:BUILD_ENV -eq "msvc") {
-           Start-FileDownload "http://servo-rust.s3.amazonaws.com/build/openssl-and-ffmpeg.zip" -ErrorAction Stop ;
-           Start-FileDownload "http://servo-rust.s3.amazonaws.com/build/ninja.zip" -ErrorAction Stop ;
-           Start-FileDownload "http://servo-rust.s3.amazonaws.com/build/MozillaBuildSetup-2.2.0.exe" -ErrorAction Stop ;
-           .\MozillaBuildSetup-2.2.0.exe /S | Out-Null ;
-           7z x openssl-and-ffmpeg.zip | Out-Null ;
-           7z x ninja.zip | Out-Null ;
-           Copy-Item C:\mozilla-build\yasm\yasm.exe C:\mozilla-build\msys\bin ;
-           Copy-Item C:\mozilla-build\mozmake\mozmake.exe C:\mozilla-build\msys\bin ;
-           $env:MOZTOOLS_PATH="C:\mozilla-build\msys\bin" ;
-           $env:NATIVE_WIN32_PYTHON="C:/Python27/python.exe" ;
-           $env:PATH="$pwd\ninja;$env:PATH" ;
-           $env:OPENSSL_INCLUDE_DIR="$pwd\openssl-bin\openssl-1.0.1t-vs2015\include" ;
-           $env:OPENSSL_LIB_DIR="$pwd\openssl-bin\openssl-1.0.1t-vs2015\lib64" ;
-           $env:OPENSSL_LIBS="ssleay32MD:libeay32MD" ;
-           $env:FFMPEG_INCLUDE_DIR="$pwd\ffmpeg-bin\include" ;
-           $env:FFMPEG_LIB_DIR="$pwd\ffmpeg-bin\lib" ;
-           $env:FFMPEG_LIBS="avformat:avcodec:avutil" ;
-        }'
-  - if %BUILD_ENV%==msvc call "C:\Program Files (x86)\Microsoft Visual Studio 14.0\VC\bin\amd64\vcvars64.bat"
+#  - ps: 'if ($env:BUILD_ENV -eq "msvc") {
```

Comment:
``bootstrap-msvc`` is replaced by ``bootstrap`` in newer commit, which also support gnu builds, but for msvc build, command is being run in ``ensure_bootstrapped``. But i did that for gnu builds.